main/tests/server/src/com/google/refine/exporters/TsvExporterTests.java

```diff
@@ -170,14 +169,31 @@ public void exportTsvWithQuote(){
         }
 
         Assert.assertEquals(writer.toString(), "column0\tcolumn1\tcolumn2\n" +
-                                               "row0cell0\trow0cell1\trow0cell2\n" +
-                                               "row1cell0\t\"line has \"\"quote\"\"\"\trow1cell2\n" +
-                                               "row2cell0\trow2cell1\trow2cell2\n");
+                "row0cell0\trow0cell1\trow0cell2\n" +
+                "row1cell0\tline has \"quote\"\trow1cell2\n" +
+                "row2cell0\trow2cell1\trow2cell2\n");
+    }
+   // CS427 Issue Link: https://github.com/OpenRefine/OpenRefine/issues/2071
+    @Test
+    public void exportTsvWithOutsideQuotes() {
+        CreateGrid(3, 3);
+
+        project.rows.get(1).cells.set(1, new Cell("\"whole line is quoted\"", null));
+        try {
+            SUT.export(project, options, engine, writer);
+        } catch (IOException e) {
+            Assert.fail();
+        }
+
+        Assert.assertEquals(writer.toString(), "column0\tcolumn1\tcolumn2\n" +
+                "row0cell0\trow0cell1\trow0cell2\n" +
+                "row1cell0\t\"whole line is quoted\"\trow1cell2\n" +
+                "row2cell0\trow2cell1\trow2cell2\n");
```

Comment:
This test case worries me a bit. If we have a cell whose string is already surrounded by quotes and export it like you specify above, then I bet most parsers (including ours) will remove those leading and trailing quotes at parsing time, by understanding them as being part of the format?